File: hoosong0235/Bronze/4153. 직각삼각형/직각삼각형.py
while (True):
    raw = input().split(" ")
    if (raw == ["0", "0", "0"]):
        break
    else:
        for i in range(3):
            raw[i] = int(raw[i])
        raw.sort()
        if (raw[0]**2+raw[1]**2==raw[2]**2):
            print("right")
        else:
            print("wrong")

File: hoosong0235c/Bronze/4153. 직각삼각형/직각삼각형.c
#include <stdio.h>

int main()
{
    int x, y, z;
    scanf("%i %i %i", &x, &y, &z);

    while (x || y || z)
    {
        int x2 = x * x, y2 = y * y, z2 = z * z;

        if ((x2 == y2 + z2) || (y2 == x2 + z2) || (z2 == x2 + y2))
            printf("right\n");
        else
            printf("wrong\n");

        scanf("%i %i %i", &x, &y, &z);
    }
}